scripts/setup.sh

```diff
@@ -0,0 +1,313 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+function main() {
+	cat <<EOF
+===============================================================================
+amiapp-reactnative setup checklist
+
+Follow the steps in to setup dev environment to be able to build and run the
+amiapp-reactnative app in the simulator or on a phone.
+You can follow along and manually run all the steps, but some do offer you the
+option to run the commands for you as well.
+===============================================================================
+EOF
+
+	step 'Install Xcode from the App Store or the Self Service app'
+
+	step 'Set xcode command line tools
+- Open Xcode
+- In the menu bar select Xcode -> Settings
+- Go to the Locations tab
+- Ensure Command Line Tools are set under the dropdown
+	- If it mentions "No Xcode Selected" just pick the latest version from the dropdown
+https://stackoverflow.com/questions/50404109/unable-to-locate-xcode-please-make-sure-to-have-xcode-installed-on-your-machine/51246596#51246596'
+
+	step 'Install Android Studio from their website or using homebrew by running:
+	brew install --cask android-studio' run_install_android_studio
+
+	step 'Set environment variables for java and android tools by running:
+	export JAVA_HOME="/Applications/Android Studio.app/Contents/jbr/Contents/Home"
+	export ANDROID_SDK_ROOT="$HOME/Library/Android/sdk"
+	export PATH="$ANDROID_SDK_ROOT/platform-tools:$PATH"
+	export PATH="$ANDROID_SDK_ROOT/tools:$PATH"
```

Comment:
We need to add this step for setting up `adb` in our `DEVELOPMENT` document as well, I can do that in a separate PR later.

Reply:
Yes, please!